sample/.github/workflows/test.yml

```diff
@@ -22,6 +57,12 @@ jobs:
     runs-on: macos-latest
     timeout-minutes: 30
     steps:
+      - name: Set up JDK 17
```

Comment:
Same here, just like setting up Ruby [here](https://github.com/nimblehq/kmm-templates/pull/16/files#diff-5e07d95bd5365e72e833c898fbf6182425895a3c755936bfea3ae5a179dbc5d3R41).

Reply:
@AVI5HEK This sample files will be regenerate by bot. So we don't need to update it 🙏🏻